### PR TITLE
Fix Baichuan2 prompt format

### PR DESCRIPTION
**Pull request by NovTi**
## Description

Fix Baichuan2 prompt format referred from [link1](https://github.com/baichuan-inc/Baichuan2/issues/227) and [link2](https://huggingface.co/baichuan-inc/Baichuan2-7B-Chat/blob/main/generation_utils.py#L7-L49).

### CODE REVIEW

**Review by hkvision — APPROVED** [review]
Also need to update the results in the README?

**Review by plusbang — COMMENTED** [review]
Please also update the prompt format in https://github.com/intel-analytics/BigDL/blob/main/python/llm/example/GPU/PyTorch-Models/Model/baichuan2/generate.py#L25

**Comment by NovTi** [comment]
> Please also update the prompt format in https://github.com/intel-analytics/BigDL/blob/main/python/llm/example/GPU/PyTorch-Models/Model/baichuan2/generate.py#L25



> Also need to update the results in the README?

Have updated README and `PyTorch-Models/Model/baichuan2/generate.py`

File: python/llm/example/CPU/HF-Transformers-AutoModels/Model/baichuan2/generate.py [hunk 1]
```diff
@@ -22,8 +22,11 @@
 from bigdl.llm.transformers import AutoModelForCausalLM
 from transformers import AutoTokenizer
 
-# you could tune the prompt based on your own model,
-BAICHUAN_PROMPT_FORMAT = "<human>{prompt} <bot>"
+# prompt format referred from https://github.com/baichuan-inc/Baichuan2/issues/227 
+# and https://huggingface.co/baichuan-inc/Baichuan2-7B-Chat/blob/main/generation_utils.py#L7-L49
+# Though this is the official prompt format, we found it has problem generating English answers
+# If you want to ask English questions, we recommend you to change the prompt format
```
Comment (hkvision) [inline python/llm/example/CPU/HF-Transformers-AutoModels/Model/baichuan2/generate.py:28]: For English prompt, you are recommended to change the prompt format.

**Pull request merged**